tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -28,7 +27,7 @@
 
 
 @skipUnless(HAS_GEOS, "Geos is required.")
-class GEOSTest(unittest.TestCase, TestDataMixin):
+class GEOSTest(TestCase, TestDataMixin):
```

Comment:
Use a `SimpleTestCase` instead as it provides `assertRaisesMessage` and avoids wrapping each test case in a transaction which is not required here.